recipes/cf-python/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cf-python" %}
+{% set version = "3.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d7622d60357811599352a31cce9d8055ed82fc3710db498ee2d376d52726d50a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
```

Comment:
```suggestion
requirements:
  build:
    - {{ compiler('c') }}
    - make
```

Reply:
They do import `Extension` but I believe that was a mistake b/c it is not used anywhere and I could not find any compiled extension in the source code. I'll check again though.

The problem here seems related to `cfunits`.

Reply:
Oh wait. I missed your comment above. Let me check it again.